addons/website_sale_comparison_wishlist/static/src/js/wishlist.js

```diff
@@ -4,21 +4,22 @@ import publicWidget from "web.public.widget";
 import "website_sale_comparison.comparison";
 
 publicWidget.registry.ProductComparison.include({
-  events: _.extend(
-    {},
-    publicWidget.registry.ProductComparison.prototype.events,
-    {
-      "click .wishlist-section .o_add_to_compare": "_onClickCompare",
-    }
-  ),
-  /**
-   * @private
-   * @param {Event} ev
-   */
-  _onClickCompare: function (ev) {
-    const $el = $(ev.currentTarget);
-    let productID = $el.data('product-id');
-    productID = parseInt(productID, 10);
-    this.productComparison._addNewProducts(productID);
-  },
+    events: _.extend({}, publicWidget.registry.ProductComparison.prototype.events, {
+        "click .wishlist-section .o_add_to_compare": "_onClickCompare",
+    }),
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onClickCompare: function (ev) {
+        const $el = $(ev.currentTarget);
+        let productID = $el.data('product-id');
+        productID = parseInt(productID, 10);
```

Comment:
- One line
- Could this even avoid using jQuery or is it really jQuery data? otherwise:
```javascript
const productID = parseInt(ev.currentTarget.dataset.productId, 10);
```